<ui version="4.0" >
 <class>MatisseNetworkForm</class>
 <widget class="QWidget" name="MatisseNetworkForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>30</y>
     <width>601</width>
     <height>571</height>
    </rect>
   </property>
   <property name="currentIndex" >
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab" >
    <attribute name="title" >
     <string>Parameters</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>40</y>
       <width>562</width>
       <height>120</height>
      </rect>
     </property>
     <property name="title" >
      <string>Thermal load</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label_1" >
          <property name="text" >
           <string>Height of canisters network (in cells)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="lineEdit_hreso1" >
          <property name="maximumSize" >
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="labelTo_1" >
          <property name="text" >
           <string>----></string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <widget class="QLineEdit" name="lineEdit2_1" />
        </item>
        <item row="0" column="4" >
         <widget class="QLabel" name="labelUnit_1" >
          <property name="text" >
           <string>m</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Height max of double jacketed wells (in cells)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="lineEdit_hreso2" >
          <property name="maximumSize" >
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QLabel" name="labelTo_2" >
          <property name="text" >
           <string>----></string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" >
         <widget class="QLineEdit" name="lineEdit2_2" />
        </item>
        <item row="1" column="4" >
         <widget class="QLabel" name="labelUnit_2" >
          <property name="text" >
           <string>m</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Height min of double jacketed wells (in cells)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" >
         <widget class="QLabel" name="labelTo_3" >
          <property name="text" >
           <string>----></string>
          </property>
         </widget>
        </item>
        <item row="2" column="3" >
         <widget class="QLineEdit" name="lineEdit2_3" />
        </item>
        <item row="2" column="4" >
         <widget class="QLabel" name="labelUnit_3" >
          <property name="text" >
           <string>m</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLineEdit" name="lineEdit_hplen" >
          <property name="maximumSize" >
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2" >
    <attribute name="title" >
     <string>Canisters network</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>20</y>
       <width>551</width>
       <height>501</height>
      </rect>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Line</string>
      </attribute>
      <widget class="MatisseCustomView" native="1" name="widgetLine" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>10</y>
         <width>521</width>
         <height>451</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4" >
      <attribute name="title" >
       <string>Row</string>
      </attribute>
      <widget class="MatisseCustomView" native="1" name="widgetRow" >
       <property name="geometry" >
        <rect>
         <x>20</x>
         <y>20</y>
         <width>511</width>
         <height>441</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MatisseCustomView</class>
   <extends>QWidget</extends>
   <header>MatisseCustomView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
